lib/ace/ext/papeeria/highlighter.coffee

```diff
@@ -10,37 +10,46 @@ define((require, exports, module) ->
             session.removeMarker(session.$bracketHighlightRight)
             session.$bracketHighlightLeft = null
             session.$bracketHighlightRight = null
+            return
         if !pos.mismatch
             rangeLeft = new Range(pos.left.row, pos.left.column, pos.left.row, pos.left.column + 1)
             rangeRight = new Range(pos.right.row, pos.right.column, pos.right.row, pos.right.column + 1)
             session.$bracketHighlightLeft = session.addMarker(rangeLeft, "ace_bracket", "text")
             session.$bracketHighlightRight = session.addMarker(rangeRight, "ace_bracket", "text")
         else
-            if pos.left && pos.right 
+            if pos.left && pos.right
                 range = new Range(pos.left.row, pos.left.column, pos.right.row, pos.right.column + 1)
                 session.$bracketHighlightLeft = session.addMarker(range, "ace_error-marker", "text")
-
             if pos.left && !pos.right
                 rangeLeft = new Range(pos.left.row, pos.left.column, Infinity, Infinity)
                 session.$bracketHighlightLeft = session.addMarker(rangeLeft, "ace_error-marker", "text")
-
             if pos.right && !pos.left
                 rangeRight = new Range(0, 0, pos.right.row, pos.right.column + 1)
                 session.$bracketHighlightRight = session.addMarker(rangeRight, "ace_error-marker", "text")
+        session.$positionOfHighlight = pos
```

Comment:
rename it to $highlightRange. Because it is not a position, it is a range. And names like 'thisOfThat' sound clumsy: in English it is natural to use sequences like "editor highlight position" which are  read right-to-left: "position of the highlight of the editor"